app/views/developers/registrations/new.html.erb

```diff
@@ -1,79 +1,83 @@
-<%= stylesheet_link_tag 'developers' %>
```

Comment:
Input fields width is shrunk when there's an error. Is this intentional? It's affecting every form.

![image](https://user-images.githubusercontent.com/51726526/122298209-db24ad00-ced2-11eb-8c0e-d30e7ff6bf38.png)

Reply:
Fixed.